Client/Frontend/Browser/BrowserTrayAnimators.swift

```diff
@@ -94,8 +94,29 @@ private extension TrayToBrowserAnimator {
             bvc.topToolbar.isTransitioning = false
             bvc.updateTabsBarVisibility()
             transitionContext.completeTransition(true)
+            
+            self.WKWebViewPDFNotRenderingBugFix(for: bvc)
         })
     }
+    
+    /// Fixes a Bug in iOS 12.2 where:
```

Comment:
Does this happen on this iOS version only?

Reply:
I am not able to verify it on iOS 12.1 or 12.3+ as I don't have those devices.. However, it happens in Simulator 12.3+.

Reply:
Thanks for explanation, could you add a rdar link or bug number to the comment?

Reply:
@iccub Added to the comment the links for WebKit bug tracker and for StackOverflow where a Firefox Developer also having the same issue but solved it worse by reloading the entire page and redrawing the PDF :(